charts/rancher-portshift-operator/0.1.0/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: portshift-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "portshift-operator.name" . }}
+    chart: {{ template "portshift-operator.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Why not our regular permissions?

Reply:
This it the permissions for the operator, it's like the "upgrader" need to install the full bundle.